src/pages/patientView/PatientViewPage.tsx

```diff
@@ -47,6 +47,14 @@ export interface IPatientViewPageProps {
     clinicalDataStatus?: RequestStatus;
 }
 
+export function updatePatientPageTitle(ids: { patientId?:string, sampleId?:string }){
```

Comment:
bike shedding is when its easier to wrap your head around a small thing than a bigger thing so you take a big stand on a small thing :D